app/helpers/discussions_helper.rb

```diff
@@ -23,8 +23,9 @@ def user_discussions_icon(text)
     fixed_fa_icon 'comment', text: text
   end
 
-  def discussions_link(item, path, html_options = nil, organization = Organization.current)
-    link_to item, path, html_options if organization.forum_enabled?
+  def discussions_link(item, path, html_options = nil)
+    return unless current_user&.can_discuss_here?
```

Comment:
I don't have a strong position here, but is there any reason to prefer this validation style to the other?

Reply:
I don't have a strong position either, but generally we are using this style when it comes to permission related actions, so it seemed more natural to do it this way